docs/02-usage.md

```diff
@@ -18,9 +19,9 @@ Let the music play!
 
 balenaSound supports multiple modes of operation described below:
 
-- Multi-room mode
-- Multi-room client mode
-- Standalone
+* Multi-room mode
```

Comment:
any reason to switch to `*` from `-`? since in the above file it was changed from `*` to `-`

Reply:
my ide was telling me to do that. Imma check that again later. Maybe knowing the difference will explain it.

Reply:
They both do the same work. Maybe the linter was configured to use the later and the IDE was set to former?